homeassistant/components/gtfs/sensor.py

```diff
@@ -357,8 +359,8 @@ def get_next_departure(
     yesterday_start = today_start = tomorrow_start = None
     yesterday_last = today_last = ""
 
-    for _row in result:
-        row = _row._mapping
+    for row_cursor in result:
+        row = row_cursor._asdict()
```

Comment:
This is still accessing a protected method, which should not be used.

Reply:
as per my comment, with me pylint accepted it where it did not the previous one…and I have reached my limits on understanding hoe to alternatively donthid

Reply:
But this code doesn't need to do that. It can just as easily leave `row` as is and change all the `row["yesterday"]` to `row.yesterday` ...etc... instead

Reply:
But I guess that won't work for https://github.com/home-assistant/core/pull/89175/files#diff-e1c37923d5176b56acc08bb3063778076c94c5e02ec4c69d92ecf28c1590589eR383

Reply:
So I think you are stuck with `_asdict()`

Reply:
Or rewrite the lot ... which very honestly I am not interested in. I'd rather fix the other issues as datetime or even add some things that the SQL is not covering than going into (for me, sorry) nitty-gritty python stuff....I know I know, your comments have a reason too :)